std/array.d

```diff
@@ -380,6 +380,95 @@ if (isNarrowString!String)
     )));
 }
 
+/**
+ * Returns a static array from the stack with length of s and initializes
+ * it with copies of the elements of range $(D r). The remainder of the
+ * array is default-initialized.
+ *
```

Comment:
The documentation should add that this is recommended particularly for nontrivial initialization of static arrays during compilation.